Yakymenko_Task_3.py

```diff
@@ -0,0 +1,16 @@
+# Multiparadigmatic  programming languages:Task_3
+# Yakymenko Semen:IKM-221a
+
+
+import math
+
+print('Multiparadigmatic  programming languages:Task_3\nSemen Yakymenko:IKM-221a')
+
+TEMPLATE = 'Please Enter {} : '
+x = int(input(TEMPLATE.format('x')))
+z = int(input(TEMPLATE.format('z')))
+if z != 2:
+    res = (x * math.atan(z)) + math.exp((x + 3) / (z - 2))
+    print('Result is:', res)
+else:
+    print('Error,your function will not work,type something else')
```

Comment:
```suggestion
# Multiparadigmatic  programming languages:Task_3
# Yakymenko Semen:IKM-221a
import math

print('Multiparadigmatic  programming languages:Task_3\nSemen Yakymenko:IKM-221a')

TEMPLATE = 'Please Enter {} : '
x = int(input(TEMPLATE.format('x')))
z = int(input(TEMPLATE.format('z')))

if z != 2:
    res = (x * math.atan(z)) + math.exp((x + 3) / (z - 2))
    print(f'Result is: {res}')
else:
    print('Error,your function will not work,type something else')
```

Reply:
Your code looks much better now. but please take care about putting blank lines they should clearly represent logical block of your program.
ask me during the class for detailed explanation

Reply:
also consider a case of using f-stirng
    print(f'Result is: {res}')
https://realpython.com/python-f-strings/